.circleci/config.yml

```diff
@@ -95,9 +108,17 @@ jobs:
       - run:
           name: Create test DB
           command: cd spec/decidim_dummy_app && RAILS_ENV=test bundle exec rake db:create db:schema:load
+      - run: cc-test-reporter before-build
       - run:
           name: Run core RSpec
-          command: cd decidim-core && rake
+          command: cd decidim-core && bundle exec rake
```

Comment:
I think @josepjaume intentionally changed this (and the other equivalent places) to not run through `bundler`. I guess there's no possible version ambiguity since the image will have a single bundle installed.